README.md

```diff
@@ -643,340 +531,322 @@ terragrunt = {
         "refresh"
       ]
     }
+  }
+}
+```
```

Comment:
I had to read this a few times to get it. I think the word "switch" through me. Maybe reword? Also, I didn't immediately make the connection you were still using a list of strings but had the first and second string as arg and arg-value.